.github/workflows/automation-pull-subtrees.yml

```diff
@@ -63,6 +63,13 @@ jobs:
       - name: Fetch submodules
         run: git submodule update --init --depth=1 --jobs=$(nproc)
 
+      # A component depends on 2024 Edition, which requires at least Rust v1.85,
+      # but the runner has Rust 1.84.
+      # TODO: Remove this step once Rust v1.85 is available in the runner
+      # https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#rust-tools
+      - name: Use 2024 edition Rust toolchain
+        run: rustup update
```

Comment:
I would scope this to `rustup update stable`. If for whatever reason GitHub starts including nightly I don't want to spend time updating it.